examples/lusid/corporate-actions-source/create-corporate-action-source.sql

```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
+-- 1. In this query, we run an ETL process on a corporate action 
+-- source.
+-- 2. Firstly, create a corporate action source and assign it to 
+-- a data table.
+-- 3. We then write this table to LUSID using the provider.
+-- ===============================================================
+
+-- Defining corporate action scope and code.
+
+@@corporateActionSourceScope = select 'luminesce-examples';
+@@corporateActionSourceCode = select 'uk-equity';
+
+-- Define a table containing sample information.
+
+@corporate_action_source_table =
+
+select  "Sample Action Source" as DisplayName,
+        "Source description" as Description,
+        @@corporateActionSourceScope as CorporateActionSourceScope,
+        @@corporateActionSourceCode as CorporateActionSourceCode,
+        "instrument-scope" as InstrumentScope,
```

Comment:
This should be InstrumentScopes plural and is defined as "The list of instrument scopes used as the scope resolution strategy when resolving instruments of upserted corporate actions.", I would leave this blank though as it is optional.